bin/install.sh

```diff
@@ -69,14 +69,15 @@ for package in "${packages[@]}"; do
         # install the package
         sudo dnf install terraform -y
     else
-        echo "package is not installed and not available."
+        echo "package $package is not installed and not available."
     fi
 
 done
 
 
 # pip install requirements
-pip3 install -U boto boto3 yq --user
+pip3 install boto boto3 --user
+pip3 install 'yq==2.10.0' 
```

Comment:
Is there a reason why `--user` was dropped for this pip install?

Reply:
No reason, good catch. Fixed in subsequent commit.